WindowState/library/src/main/java/com/microsoft/device/dualscreen/windowstate/WindowSizeClass.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.windowstate
+
+import androidx.annotation.VisibleForTesting
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+enum class WindowSizeClass { COMPACT, MEDIUM, EXPANDED }
+
+/**
+ * Calculates size class for a given dimension
+ *
+ * @param dimenDp: size of dimension in Dp
+ * @param dimen: which dimension is being measured (width or height)
+ */
+@VisibleForTesting
```

Comment:
I think this annotation is no longer needed, it shows me a warning in `WindowState.kt`